controllers/resource/resource_generator.go

```diff
@@ -130,14 +130,18 @@ func (r *resourceGenerator) GetPodTemplate(doguResource *k8sv1.Dogu, dogu *core.
 
 	allLabels := GetAppLabel().Add(doguResource.GetPodLabels())
 
+	// Avoid env vars like <service_name>_PORT="tcp://<ip>:<port>" because they could override regular dogu env vars.
```

Comment:
That is a proper comment, I like that it explains things that might be hidden from developers unaware of the way service links work. 

Anyhow, I fail to grasp how any changes from the dogu will modify this behaviour.

Reply:
Okay, I misunderstood the (potentially) alternative of enabling service links. Now I understand there will be no support for service links, and that is good as well.